### PR TITLE
Q1-2023 community board update

### PR DESCRIPTION
**Pull request by thatdocslady**
----
:books: Documentation preview :books:: https://writethedocs-www--1906.org.readthedocs.build/

### CODE REVIEW

**Comment by thatdocslady** [comment]
Direct link to preview: https://writethedocs-www--1906.org.readthedocs.build/blog/2023-Q1-community-board/

**Review by ericholscher — APPROVED** [review]
This looks great to me! What's out distribution plan? Slack/Buffer, and mentioned in the newsletter, but no email for this post specific?

File: docs/blog/2023-Q1-community-board.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+In case you want a refresher on how the board was formed and who is a member, here is the TL;DR:
+
+The `Community Board <https://www.writethedocs.org/team/#community-board>`_ exists to curate the overall strategic direction of the community.
+
+The members are the team leads of each of the Write the Docs team, as well as the conference chairs of our various conferences:
+
+* Mikey Ariel (CoreOps, Brand and design, board chair)
+* Eric Holscher (CoreOps, Portland chair)
+* Samuel Wright (CoreOps, Infrastructure)
+* Sasha Romijn (CoreOps, Support)
```
Comment (plaindocs) [inline docs/blog/2023-Q1-community-board.rst]:
```suggestion
* Sasha Romijn (CoreOps, Support, Atlantic Chair)
```

File: docs/blog/2023-Q1-community-board.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+   :author: Mikey Ariel
+
+Write the Docs Community Board Quarterly Update Q1-2023
+=========================================================
+
+Greetings, documentarians! Mikey Ariel here, chairperson of the newly formed community board and former head organizer of the Prague conference. 
+
+As you might have heard, at the end of 2022 I stepped down from my role as conference chair and transitioned to the new role of community board chair. The board meets quarterly during the first month of the quarter, and is comprised of each team lead in the WTD organization. 
+
+We hope that this group will be able to discuss and drive more long-term strategic initiatives and collaboration for the community.
```
Comment (plaindocs) [inline docs/blog/2023-Q1-community-board.rst]: Maybe make this more concrete than 'hope'?

**Review by plaindocs — APPROVED** [review]

**Comment by plaindocs** [comment]
@ericholscher sounds good

File: docs/blog/2023-Q1-community-board.rst (hunk 2 above)
Comment (thatdocslady) [inline docs/blog/2023-Q1-community-board.rst]:
```suggestion
Our main goal with this group is to discuss and drive more long-term strategic initiatives and collaboration for the community. 
```

**Comment by thatdocslady** [comment]
@plaindocs I'm guessing that the CI fail is related to the spellcheck PR you submitted? I don't think I'm qualified to review that one but maybe I can merge this PR without it?

**Comment by plaindocs** [comment]
That's a fix yes. You could or you could just wait.

**Pull request merged**